.gitignore-example-from-docusaurus

```diff
@@ -0,0 +1,13 @@
+.DS_Store
```

Comment:
Hey @mmBabol Can you please merge this file with the one for our project. Also, remove this file kindly.
Thanks